pkg/controller/container-runtime-config/helpers.go

```diff
@@ -954,3 +1006,67 @@ func generateSigstoreRegistriesdConfig(clusterScopePolicies map[string]signature
 	}
 	return data, nil
 }
+
+func addScopeMirrorsSigstoreRegistriesdConfig(registriesDockerConfig map[string]dockerConfig, scope string, reg *sysregistriesv2.Registry, sigstoreAttachment dockerConfig) error {
+	if reg == nil {
+		return nil
+	}
+
+	dummyDigest := "@sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+	dummyTag := ":aaa"
+	dummyPath := "/a"
+	dummyPrefix := "matched"
+
+	if strings.HasPrefix(scope, "*.") {
+		scope = strings.Replace(scope, "*", dummyPrefix, 1)
+	}
+
+	scopeRef, err := reference.Parse(scope)
```

Comment:
[As a general rule, prefer `ParseNormalizedNamed`. It shouldn’t make a difference for valid scopes, … but do we actually validate that? `IsValidRegistriesConfScope` seems rather loose, and I’m not sure that there is any other location.

I’m … worried what happens in the “scope == host name” case — AFAICS `reference.Parse` turns that into `"[nothing]/$input"`. But it might work well enough.

OTOH with `ParseNormalizedNamed`, the `dummyPath` part would really have to happen before this parsing. See elsewhere.]

Reply:
On second thought, `ParseNormalizedNamed("docker.io/library")` would produce an unwanted result (or we would have to work around that even more); `reference.Parse()` is really better here.